packages/account/src/Components/form-body-section/form-body-section.tsx

```diff
@@ -6,16 +6,37 @@ import { PlatformContext, isMobile } from '@deriv/shared';
 export type TFormBodySection = {
     has_side_note?: boolean;
     side_note?: string | React.ReactElement;
+    is_reversed?: boolean;
 };
 
-const FormBodySection = ({ children, has_side_note, side_note }: React.PropsWithChildren<TFormBodySection>) => {
+/**
+ *  A component that renders a form body section with an optional side note.
+ *
+ *  This component renders a section of a form body. It can optionally display a side note
+ *  and allows reversing the order of the side note and the main content.
+ *
+ *  @param {TFormBodySection} props - The properties of the FormBodySection component.
+ *  @param {boolean} props.has_side_note - Whether to display a side note.
+ *  @param {string | React.ReactElement} props.side_note - The content of the side note.
+ *  @param {boolean} props.is_reversed - Whether to reverse the order of the side note and the content.
+ *  @returns {JSX.Element} A React component that renders a form body section.
+ */
```

Comment:
In my opinion, This is a bit too much, especially for components that are not generic. You can ignore this comment 🙇🏻

Reply:
Yes, and I thought about this as well. Just wanted to get the opinion on this. 
We can reduce this to just doc the type properties with just a bit of information about the prop?